lib/sensu/server/process.rb

```diff
@@ -1442,6 +1445,9 @@ def update_server_registry
               }
             },
             :sensu => sensu,
+            :tessen => {
```

Comment:
The Sensu server registry data now includes if Tessen is enabled, this information is available via API /info 👍